test/013_oracles_composite.ts

```diff
@@ -82,10 +83,14 @@ describe('Oracles - Composite', function () {
     await compositeMultiOracle.setPath(DAI, USDC, [ETH])
```

Comment:
`setPath` and `setSource` are actually untested. Please do so, including testing that only authorized users can execute them.

Reply:
👍